dbms/src/Databases/DatabaseOrdinary.h

```diff
@@ -66,16 +113,25 @@ class DatabaseOrdinary : public DatabaseWithOwnTablesBase
 
     void drop() override;
 
-    void shutdown() override;
```

Comment:
Because [implementaion of that method](https://github.com/yandex/ClickHouse/blob/master/dbms/src/Databases/DatabaseOrdinary.cpp#L493) is just copypaste of base class [method](https://github.com/yandex/ClickHouse/blob/master/dbms/src/Databases/DatabasesCommon.cpp#L142).
I thought it is meaningless but we can leave it if you want.

Reply:
Ok, I will return this code soon

Reply:
I've removed this deletion.